demo/demo-schema/src/main/java/org/apache/servicecomb/demo/produceprocessor/ProduceAppXmlProcessor.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.servicecomb.demo.produceprocessor;
```

Comment:
Maybe we can support XML as a formal encoding. But before that, we should address some common vulnerable as shown in https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Prevention_Cheat_Sheet

Reply:
done